public/css/style.css

```diff
@@ -7,9 +7,10 @@
       "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Comment:
we are using Tailwind. this means we have classes in html in hbs files, no particular reason to have additional CSS